nengo/tests/test_processes.py

```diff
@@ -99,7 +100,7 @@ def test_gaussian_whitenoise(Simulator, rms, seed, plt):
 def test_whitesignal_rms(Simulator, rms, seed, plt):
     t = 1.
     d = 500
-    process = WhiteSignal(t, rms=rms)
+    process = WhiteSignal(t, high=500, rms=rms)
```

Comment:
I also like the added readability. But also curious, @tbekolay where can I find info on that about 3.5? I tried a couple Google searches, but evidently I'm not using the right keywords (pun intended).